### PR TITLE
Code Review Practice

### PR DESCRIPTION
**Pull request by altontonn**
### Code Review 
This is a code review repo for my partners to check best JavaScript  and HTML practice

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+enterKey.addEventListener('keyup', (e) => {
+  if (e.keyCode === 13) {
+    addTask();
+  }
+});
+
+window.addEventListener('DOMContentLoaded', () => {
+  displayTodo();
+  addTask();
+});
```
Comment (Haadiiii) [inline src/index.js:81]: good job by sticking with ES6, 👍  but you have to create separate `JS` files in the modules folder for each function and create variables. it'll look so awesome after doing this 👍  😉

**Review by Haadiiii — CHANGES_REQUESTED** [review]
Hello @altontonn .

## Changes Required ♻️ 
good jobe so far you done really greate but there is something which requires changes 👍 🐧

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+<body>
+    <div class="header">
+        <h1>Today's To Do</h1>
+        <i id="refersh" class="fa-solid fa-rotate icon"></i>
+    </div>
+    <div class="list">
+        <div id="add-item" class="item-container">
+            <div class="item">
+                <input id="new-item" type="text" name="to-do-task" placeholder="Add to your list ..." />
+                <i id="enter" class="fa-solid fa-arrow-turn-down arrows enter-icon"></i>
```
[the remark this answers is not in the document]
Comment (altontonn) [reply src/index.html:22]: Hi @Haadiiii, how do I create the input field dynamically?

**Review by Haadiiii — APPROVED** [review]

File: src/index.js (hunk 1 above)
Comment (Haadiiii) [inline src/index.js:81]: [OPTIONAL]; good job by sticking with ES6, 👍 but you have to create separate JS files in the modules folder for each function and create variables. it'll look so awesome after doing this 👍

**Review by Haadiiii — APPROVED** [review]
hey @altontonn 

you do really great... there nothing to say other than...... to merge the PR 👍 🐧

**Pull request merged**